lib/whitehall_importer/sync.rb

```diff
@@ -22,7 +22,6 @@ def call
       GdsApi.whitehall_export.document_migrated(
         whitehall_import.whitehall_document_id,
       )
-      MigrateAssets.call(whitehall_import)
```

Comment:
I'm not very comfortable with us deleting this line. In the unlikely event that we ever wanted to do a future migration the migrator might not realise they're missing the assets. I've added a comment to the slack thread about this.